pkg/plugin/v3/scaffolds/internal/templates/main.go

```diff
@@ -124,12 +121,9 @@ const (
 		os.Exit(1)
 	}
 `
-	// TODO(v3): loggers for the same Kind controllers from different groups use the same logger.
-	//  `.WithName("controllers").WithName(GROUP).WithName(KIND)` should be used instead. However,
-	//  this is a backwards incompatible change, and thus should be done for next project version.
-	multiGroupReconcilerSetupCodeFragment = `if err = (&%scontroller.%sReconciler{
+	multiGroupReconcilerSetupCodeFragment = `if err = (&%scontrollers.%sReconciler{
 		Client: mgr.GetClient(),
-		Log: ctrl.Log.WithName("controllers").WithName("%s"),
+		Log: ctrl.Log.WithName("controllers").WithName("%s").WithName("%s"),
```

Comment:
```suggestion
		Log: ctrl.Log.WithName("controllers").WithName("%s"),
```

Reply:
Disregard.

Reply:
Split as requested.

Reply:
Eh this can probably go in the same PR since it's related. I didn't see the TODO above this change.

Reply:
one already in another. You are right, the correct way is to have 1 pr for each change/motivation.